jcl/src/openj9.criu/share/classes/org/eclipse/openj9/criu/CRIUSupport.java

```diff
@@ -647,6 +654,7 @@ public synchronized void checkpointJVM() {
 					registerRestoreEnvVariables();
 				}
 
+				J9InternalCheckpointHookAPI.registerPostRestoreHook(RESTORE_CLEAR_INETADDRESS_CACHE_PRIORITY, "Restore clear InetAddress Cache", CRIUSupport::clearInetAddressCache);
```

Comment:
"Restore clear InetAddress Cache" -> "Clear InetAddress cache on restore"

Reply:
Changed.
Also updated the `exports` & `imports` for JDK11 - https://github.com/ibmruntimes/openj9-openjdk-jdk11/pull/661